File: user-service/src/main/java/io/memberize/userservice/web/MemberResponse.java
package io.memberize.userservice.web;

import com.fasterxml.jackson.annotation.JsonInclude;

@JsonInclude(JsonInclude.Include.NON_EMPTY)
public class MemberResponse {
	private String message;

	public MemberResponse() {
	}

	public MemberResponse(String message) {
		this.message = message;
	}

	public String getMessage() {
		return message;
	}

	public void setMessage(String message) {
		this.message = message;
	}
}

File: user-service/src/main/java/io/memberize/userservice/web/MemberResponse.kt
package io.memberize.userservice.web

import com.fasterxml.jackson.annotation.JsonInclude

@JsonInclude(JsonInclude.Include.NON_EMPTY)
class MemberResponse {
	var message: String? = null

	constructor()

	constructor(message: String) {
		this.message = message
	}
}
